packages/react/src/components/pickers/Suggestions/Suggestions.types.ts

```diff
@@ -77,6 +77,11 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
    */
   mostRecentlyUsedHeaderText?: string;
 
+  /**
+   * The icon that appears indicating to the user that they can search for more results.
+   */
+  searchForMoreIconName?: string;
```

Comment:
Typically now when adding icon customization, we just let them pass all the props instead of just the name (to avoid needing to add more APIs later)
```suggestion
  searchForMoreIconProps?: IIconProps;
```